client/blocks/happychat/connection.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import HappychatClientApi from 'happychat-client';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { getCurrentUser } from 'state/current-user/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSkills from 'state/happychat/selectors/get-skills';
+import {
+	AUTH_TYPE_WPCOM_PROXY_IFRAME,
+	ENTRY_CHAT,
+	LAYOUT_PANEL_MAX_PARENT_SIZE,
+} from 'blocks/happychat/chat-client/constants';
+
+export default ( { state, dispatch }, {
```

Comment:
Do we need the `dispatch` here? If not, we don't need to pass it in `HappychatClient` component `componentDidMount`

Reply:
I forgot to delete it when splitting in PRs, it is used in the next PR https://github.com/Automattic/wp-calypso/pull/24107/files